scrapy/utils/misc.py

```diff
@@ -184,6 +184,25 @@ def set_environ(**kwargs):
                 os.environ[k] = v
 
 
+def walk_callable(node):
+    """Similar to ``ast.walk``, but walks only function body and skips nested
+    functions defined within the node.
+    """
+    from collections import deque
+    todo = deque([node])
+    walked_func_def = False
+    while todo:
+        node = todo.popleft()
+        if isinstance(node, ast.FunctionDef):
+            if walked_func_def:
+                # don't traverse nested functions
+                continue
+            # first function def is ok
```

Comment:
```suggestion
                continue
```
These comments are unnecessary, I think their purpose is already explained the docstring.